sandboxjs/sandbox_version.json

```diff
@@ -1,4 +1,4 @@
 {
-  "release": "v6.9.5-SNAPSHOT",
-  "version": "6.9.5-SNAPSHOT"
+  "release": "v6.9.6-SNAPSHOT",
+  "version": "6.9.6-SNAPSHOT"
```

Comment:
This change needs to be done in a follow up PR once `6.9.6-SNAPSHOT` sanbox is created.